video/out/cocoa_cb_common.swift

```diff
@@ -262,6 +262,7 @@ class CocoaCB: Common, EventSubscriber {
 
         uninit()
         uninitCommon()
+        window = nil
```

Comment:
Can `shutdown()` be called again? Because if we null window here, those checks about wouldn't work anymore
```
isShuttingDown = window?.isAnimating ?? false ||
                         window?.isInFullscreen ?? false && option.vo.native_fs
        if window?.isInFullscreen ?? false && !(window?.isAnimating ?? false) {
            window?.close()
        }
if isShuttingDown { return }
```
but I don't fully follow what why this is checking all those things, so maybe it is ok. Just want to make sure.

Reply:
Ok, it is hard for me to follow. But I just wanted to make sure that we are certain reference to window won't be needed after this point. Thanks.